src/handlebars/about.handlebars

```diff
@@ -71,13 +71,12 @@
           </li>
           <li>
             <strong>Can I bundle AdoptOpenJDK binaries in my application and distribute them?</strong><br/>
-              * Hotspot based binaries, Yes. The GPLv2+CE license allows you do this.<br/>
-              * Eclipse OpenJ9 based binaries, Yes. The licenses allow for this.
+              Yes. The GPLv2+CE license allows you do this.
```

Comment:
`... allows you to do this.` perhaps? Missing `to` before `do`?

Reply:
fixed